numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1817,6 +1837,22 @@ unpack_bits(PyObject *input, int axis, PyObject *count_obj)
     in_stride = PyArray_STRIDE(new, axis);
     out_stride = PyArray_STRIDE(out, axis);
 
+
+#if NPY_BYTE_ORDER == NPY_LITTLE_ENDIAN
+     if (order == 'l') {
```

Comment:
nit: indent is off, should be 4 spaces not 5

Reply:
fixed

Reply:
removed all this code